plone/recipe/codeanalysis/__init__.py

```diff
@@ -77,6 +79,9 @@ def __init__(self, buildout, name, options):
         # CSS Lint
         self.options.setdefault('csslint', 'False')
         self.options.setdefault('csslint-bin', 'csslint')
+        # check-manifest
+        self.options.setdefault('check-manifest', 'True')
```

Comment:
@tisto being ``check-manifest`` an external package I don't mind/care if zest.releaser or whatnot is *also* including it, but I do care that all our general QA checks that we want to do are here at p.r.codeanalysis.

For me p.r.codeanalsysis is basically the one-stop shop for anything regarding QA. So in short :+1: to keep it here :)